src/System.Net.Sockets/tests/FunctionalTests/AcceptAsync.cs

```diff
@@ -118,7 +118,7 @@ public void AcceptAsync_IPv6_Success()
 
         [OuterLoop] // TODO: Issue #11345
         [Fact]
-        [PlatformSpecific(PlatformID.Windows)]
+        [PlatformSpecific(Xunit.PlatformID.Windows)]
```

Comment:
Now that #12284 has been merged, fully qualifying the enum is no longer necessary as it has been renamed `PlatformID` -> `TestPlatforms`. Sorry for the merge conflicts.